clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -128,7 +133,17 @@
     CLUSTER_AUTHORIZATION_FAILED(31,
             new ClusterAuthorizationException("Cluster authorization failed.")),
     INVALID_TIMESTAMP(32,
-            new InvalidTimestampException("The timestamp of the message is out of acceptable range."));
+            new InvalidTimestampException("The timestamp of the message is out of acceptable range.")),
+    TOPIC_ALREADY_EXISTS(33,
+            new TopicExistsException("Topic with this name already exists.")),
+    INVALID_PARTITIONS(34,
+            new InvalidPartitionsException("Number of partitions is invalid.")),
+    INVALID_REPLICATION_FACTOR(35,
+            new InvalidReplicationFactorException("Replication-factor is invalid.")),
+    INVALID_REPLICA_ASSIGNMENT(36,
+            new InvalidReplicaAssignmentException("Replica assignment is invalid.")),
+    INVALID_ENTITY_CONFIG(37,
+            new InvalidEntityConfigurationException("Entity configuration is invalid."));
```

Comment:
Could we add those to ErrorMapping.scala as well? If they are not used in scala, just add them as place holders in the comment to reserve the error number.

Also, would it be simpler to rename InvalidEntityConfigurationException to just InvalidConfigurationException?